src/profile-logic/profile-data.js

```diff
@@ -1299,7 +1304,10 @@ export function defaultThreadOrder(threads: RawThread[]): ThreadIndex[] {
     const nameB = threads[b].name;
 
     if (nameA === nameB) {
-      return a - b;
+      return (
+        threadActivityScores[b].boostedSampleScore -
+        threadActivityScores[a].boostedSampleScore
+      );
```

Comment:
If the score is the same I'd still return `a-b`.
See the last line of the comment above,

Reply:
Do we want to check the score in the last line of the function too?